apps/native-component-list/src/screens/AV/VideoPlayer.tsx

```diff
@@ -52,7 +52,9 @@ export default class VideoPlayer extends React.Component<
 
   _handleVideoMount = (ref: Video) => (this._video = ref);
 
-  _updateStateToStatus = (status: any) => this.setState(status);
+  _handlePlaybackStatusUpdate = (status: any) => this.setState(status);
```

Comment:
```suggestion
  _handlePlaybackStatusUpdate = (status: SomeMeaningfulType) => this.setState(status);
```
Maybe it would help understanding what state parameters are actually changing? 🤔

Reply:
Yeah I agree. I'll have at improving the example in a separate PR 👍

Reply:
NLC update PR #9335